utils/get_latest_copr_build

```diff
@@ -2,83 +2,90 @@
 
 import json
 import os
+import re
 import sys
 
 import copr.v3
 
 
-def exit_fail():
-    sys.stderr.write('Passed (or default) environment variables:\n')
-    for var, value in ENV_VARS.items():
-        sys.stderr.write('  {}: {}\n'.format(var, value))
-    sys.exit(1)
-
-
-def get_owner_from_config():
-    # We have just the name of project inside COPR_REPO value which is expected when the user
-    # is the owner of the project. So set the user as owner from the config file.
-    return client.config['username']
-
-
 ENV_VARS = {
     '_COPR_CONFIG': '~/.config/copr',  # Copr config file. Get it through https://<copr instance>/api/.
-    'COPR_OWNER': None,  # Owner of the Copr project
-    'COPR_PROJECT': None,  # The Copr project to search
-    'COPR_PACKAGE': None,  # Name of the package to look for. This is optional - if empty, any package in the
-                           #  project is considered.
-    'PKG_RELEASE': None,  # The release part of the pkg NEVRA string, e.g.
-                          #  0.201906041623Z.f82f863.add_missing_deps.PR231
-    'COPR_REPO': None  # An alternative to COPR_OWNER & COPR_PROJECT. This env var should hold "owner/project".
+    'COPR_OWNER': '',  # Owner of the Copr project
+    'COPR_PROJECT': '',  # The Copr project to search
+    'COPR_PACKAGE': '',  # Name of the package to look for. This is optional - if empty, any package in the
+                         # project is considered.
+    'PKG_RELEASE': '',  # The release part of the pkg NEVRA string, e.g.
+                        #  0.201906041623Z.f82f863.add_missing_deps.PR231
+    'COPR_REPO': '',  # An alternative to COPR_OWNER & COPR_PROJECT. This env var should hold "owner/project".
+    'REGEX': ''   # A more general way to search for release_id matches via regex. Generalization of PKG_RELEASE
 }
-
+# override defaults with environment variables
 for env, default in ENV_VARS.items():
     ENV_VARS[env] = os.getenv(env, default)
-
 CONFIG_PATH = os.path.expandvars(ENV_VARS['_COPR_CONFIG'])
-client = copr.v3.Client(copr.v3.config_from_file(path=CONFIG_PATH))
-ownername = ENV_VARS['COPR_OWNER']
-projectname = ENV_VARS['COPR_PROJECT']
-
-if projectname and not ownername:
-    ownername = get_owner_from_config()
```

Comment:
I have on purpose removed ownername guessing from config as it is not used in the use case I need and is imho overcomplicating thing. username != owner generally speaking.

Reply:
@fernflower originally, the script has been executed with the config of the owner, which has been shared in our team. but I agree that this change seems better to me regarding nowadays expactations and switch to public repos.